Jenkinsfile

```diff
@@ -18,6 +18,9 @@ class Globals {
     // sets the pipeline to execute all steps related to releasing the service
     static boolean release = false
 
+    // sets the pipeline to execute all steps related to create and publish artifacts
+    static boolean publishArtifacts = false
```

Comment:
Seems like this option has a large overlap with the deploy option, perhaps it would make sense to retire it once you get the deploy stage implemented

Reply:
Yes, this is a temporary addition. I didn't want to remove the deploy stage in order not to stray too far from the blueprint and to be able to recover it quickly once the deploy stage is implemented.